.github/workflows/ci.yml

```diff
@@ -47,4 +47,10 @@ jobs:
         run: go vet ./...
 
       - name: gotestsum
-        run: gotestsum --format short-verbose ./...
+        run: gotestsum --format short-verbose -- -coverprofile=cover.txt ./...
+
+      - name: Upload coverage report
+        uses: codecov/codecov-action@v1.0.2
+        with:
+          file: ./cover.txt
+          name: testcontainers-go-${{ matrix.go-version }}
```

Comment:
```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
          file: ./cover.txt
          name: testcontainers-go-${{ matrix.go-version }}
```

Reply:
Mmm, it was fixed with 0913f1b344a2284c2a3922a9454fdcb1711d1710, which uses the latest GH action